sql/src/main/java/io/crate/analyze/QuerySpec.java

```diff
@@ -44,8 +41,8 @@
     private Optional<HavingClause> having = Optional.absent();
     private List<Symbol> outputs;
     private WhereClause where = WhereClause.MATCH_ALL;
-    private Optional<Integer> limit = Optional.absent();
-    private int offset = 0;
+    private Optional<Symbol> limit = Optional.absent();
+    private Symbol offset = Literal.ZERO;
```

Comment:
hm, how about making this optional. i think also the ADD(0, 0) symbols are somehow odd